src/index.tsx

```diff
@@ -97,9 +98,14 @@ export default class InfiniteScroll extends Component<Props, State> {
       this.el.addEventListener('touchmove', this.onMove);
       this.el.addEventListener('touchend', this.onEnd);
 
-      this.el.addEventListener('mousedown', this.onStart);
-      this.el.addEventListener('mousemove', this.onMove);
-      this.el.addEventListener('mouseup', this.onEnd);
+      if (
+        this.props.pullDownToRefreshOnlyOnTouchEvents !== undefined &&
+        !this.props.pullDownToRefreshOnlyOnTouchEvents
```

Comment:
just truthy will work I guess

if (this.props.pullDownToRefreshOnlyOnTouchEvents) {
  return;
}
// add event listeners

Reply:
This will prevent the code after this if statement to run, right?